patches/chrome-browser-preferences-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/preferences/BUILD.gn b/chrome/browser/preferences/BUILD.gn
+index b520755f694fd7c82b0e643fc62d6de01d43a5d7..d7a05ac30ebdffd7faf7672328da80710f13f63f 100644
+--- a/chrome/browser/preferences/BUILD.gn
++++ b/chrome/browser/preferences/BUILD.gn
+@@ -26,6 +26,7 @@ android_library("java") {
+   ]
+   annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
+   srcjar_deps = [ ":java_pref_names_srcjar" ]
++  import("//brave/android/java/org/chromium/chrome/browser/preferences/brave_java_sources.gni") sources += brave_browser_preferences_java_sources
```

Comment:
We can move import to `brave/android/brave_java_sources.gni`, this way patch will be cleaner.

Reply:
thanks, fixed at https://github.com/brave/brave-core/pull/17466/commits/554f6a336ccc2723b32898458c0f8f77204c3f13